FrontEnd/src/components/SignUp/components/signup/signup-form/SignUpFormContent.js

```diff
@@ -61,13 +62,33 @@ export function SignUpFormContentComponent(props) {
 
   useEffect(() => {
     const companyOrStartup = isChecked.company || isChecked.startup;
-    const formIsValid = companyOrStartup && isValid
+    const formIsValid = companyOrStartup && isValid;
     setIsValid(formIsValid);
-    }, [isValid, setIsValid, isChecked.company, isChecked.startup])
+  }, [isValid, setIsValid, isChecked.company, isChecked.startup]);
 
   const onSubmit = (event) => {
-    // TODO - add submission
+    event.preventDefault();
+    const dataToSend = {
+      email: getValues("email"),
+      password: getValues("password"),
+      re_password: getValues("confirmPassword"),
+      name: getValues("name"),
+      surname: getValues("surname"),
+      company: {
+        name: getValues("companyName"),
+        is_registered: isChecked.company,
+        is_startup: isChecked.startup,
+      },
+    };
+    
+    axios({
+      method: 'post',
+      url: `http://localhost:8000/api/auth/users/`,
+      withCredentials: false,
+      data: dataToSend
+    }).then(res => console.log(res.data)).catch(error => console.log(error))
```

Comment:
Should we do smth then after registration? I know that API doesn't return a token, but should we redirect the user to the logic page?

Reply:
I suppose there's another task to add modal window. But there are some inconsistences so I left it for the next task.

Reply:
I left ToDos for the future work.